Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+---
+post_title: Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+username: farisnt@gmail.com
+Catagories: PowerShell, Windows Services
+Summary: Show how to get services information using both Windows PowerShell 5.1 and PowerShell 7
+---
+
+# Get-Services, PowerShell 7 VS Windows PowerShell 5.1
+
+**Q**: How can I get the Username, and StartType for a Windows Service?
+
+**A**: Quick answer is PowerShell 7.
+
+Microsoft is doing a great job on Powershell on each version they release,  to answer this question a simple command called `Get-Service` will be used, but there is a big update that makes getting the required information way much easier with PowerShell 7. I will show the result of this command using both **PowerShell 7** and **Windows PowerShell 5.1**.
```

Comment:
I get what you are saying, but I think you are meaning you can now use Get-Service as this information wasn't available in Windows PowerShell. You could get start type but not the user (or startuptype for that matter). 
Also, the first line needs editing. 

ALso, for all articles avoid the use of figure these, so not "will" - this helps to aid in reading clarity,

Reply:
Got it :)